src/style.css

```diff
@@ -0,0 +1,150 @@
+/* #region Font */
+@font-face {
+  font-family: "Roboto";
+  src: url(fonts/Roboto-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url(fonts/Roboto-Medium.ttf);
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url(fonts/Roboto-Bold.ttf);
+  font-weight: 700;
+}
+/* #endregion */
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  --clear-star: url(images/star.svg);
+  --golden-star: url(images/star-active.svg);
+  --blue-color: #00acdc;
+  font-family: "Roboto", sans-serif;
+  font-weight: normal;
+  color: #000;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding-left: 14px;
+  padding-right: 14px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-shadow: 0 2px 5px rgba(0, 0, 0, 0.07);
+  border-radius: 5px;
+}
+
+/* #region Product-info*/
+.product-image {
+  width: 160px;
+  height: 134px;
+  margin: 29px 5px 45px;
+  padding: 0;
+}
+
+.product-code {
+  font-size: 11px;
+  line-height: 13px;
```

Comment:
You use 
  font-size: 11px;
  line-height: 13px;

four times in your code. Let's not repeat yourself. Move it to the `body`